scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+
+const inputFileName = 'RxTerms202202.txt';
+const outputFileName = 'output.txt'
```

Comment:
Let's take the files from the command line instead of hard-coding the paths.

Reply:
Sounds good. Added

Reply:
Readme added